modules/tile-converter/src/i3s-converter/helpers/node-index-document.ts

```diff
@@ -172,6 +174,15 @@ export class NodeIndexDocument {
     return (await openJson(parentNodePath, parentNodeFileName)) as Node3DIndexDocument;
   }
 
+  /**
+   * Unload the Node data
+   */
+  private flush(): void {
+    if (this.data) {
+      this.data = null;
```

Comment:
Nit: You don't really need the check before you assign `null`? Or do we support some separate `undefined` and `null` states?